package/python-pyqt5/Config.in

```diff
@@ -10,3 +10,12 @@ config BR2_PACKAGE_PYTHON_PYQT5
 	  Python bindings for Qt 5
 
 	  http://www.riverbankcomputing.com/software/pyqt/
+
+
+if BR2_PACKAGE_PYTHON_PYQT5
+config BR2_PACKAGE_PYTHON_PYQT5_CONFIG
+	string "PyQt5 Configeration file."
```

Comment:
Config _u_ ration, unless that's the PyQt5 convention :)

Reply:
I am the type-o KING!!!